src/HotChocolate/AzureFunctions/src/HotChocolate.AzureFunctions.IsolatedProcess/HotChocolate.AzureFunctions.IsolatedProcess.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>HotChocolate.AzureFunctions.IsolatedProcess</PackageId>
+    <AssemblyName>HotChocolate.AzureFunctions.IsolatedProcess</AssemblyName>
+    <RootNamespace>HotChocolate.AzureFunctions.IsolatedProcess</RootNamespace>
+    <Description>This package contains the GraphQL AzureFunctions Isolated Process integration for Hot Chocolate.</Description>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Extensions\HotChocolateAzFuncIsolatedProcessServiceCollectionExtensions.cs" />
```

Comment:
Is this an accident?

Reply:
I never got a notification for this comment, just now seeing it :-/ 

Working on getting my build working and I'll validate, but def. seems like an accident...

Reply:
Ok this is resolved, it was def. an accident somehow. ✅